static/src/stylesheets/module/facia-garnett/_pillars.scss

```diff
@@ -62,10 +66,22 @@ $pillars: (
         headlineIcon: $lifestyle-garnett-main-1,
         metaText: $garnett-neutral-2,
         cutoutBackground: $lifestyle-garnett-feature,
-        invertedKicker: $lifestyle-garnett-media-main-1
+        invertedKicker: $lifestyle-garnett-media-main-1,
+        analysisUnderline: $lifestyle-garnett-analysis
     )
 );
 
+@mixin garnett-underline($colour, $line-height) {
```

Comment:
@regiskuckaertz and me have spoken and agree that this mixin would be best located in a `_fc-item--type-analysis.scss` file as it contains layout info specific to analysis types.